### ISSUE TITLE
Consider ordering list of sensors alphabetically

### ISSUE THREAD

**Issue by marcnause** [fossasia/pslab-android#2597]
**Actual Behaviour**

The list of sensors has no obvious order.

**Expected Behaviour**

List of sensors is ordered alphabetically, 

1. Open app
2. Tap "Sensors"
3. Tap "AUTOSCAN"

**Screenshots of the issue**

<img src="https://github.com/user-attachments/assets/03d948dd-b57e-4482-88ac-609efbac1eab" width=400 />

**Would you like to work on the issue?**

Maybe

**Comment by samruddhi-Rahegaonkar** [comment]
Yup! I will do this. Assign to me please! @marcnause we can simply use `Collections.sort(dataName);`.

**Comment by samruddhi-Rahegaonkar** [comment]
done. @marcnause 
<img width="1280" alt="Screenshot 2024-12-27 at 8 51 46 PM" src="https://github.com/user-attachments/assets/e99ea5aa-4360-445d-9e07-e218413c4b88" />
Should I do PR ?
